ipymd/core/format_manager.py

```diff
@@ -37,15 +39,27 @@ class FormatManager(LoggingConfigurable):
     # The name of the setup_tools entry point group to use in setup.py
     entry_point_group = "ipymd.format"
 
+    # The name of the default kernel: if left blank, assume native (pythonX)
+    # won't store kernelspec/language_info unless forced
+    # TODO: where does this get set but by the ContentsManager?
+    default_kernel_name = Unicode(config=True)
+
+    # don't strip any metadata
```

Comment:
again, really minor detail, but could we have the capital in uppercase for all docstrings and comments (just for consistency...) ?